_posts/2023-05-24-llama-tensorflow-keras/llama-tensorflow-keras.Rmd

```diff
@@ -0,0 +1,1198 @@
+---
+title: "LLaMA in R with Keras and TensorFlow"
+description: >
+  Walk-through an implementation of LLaMA in R
```

Comment:
Hesitating to make suggestions to a "much more native" speaker, but this sounds a bit strange to me... Maybe add "of", or make a complete sentence of it? "This post walks ..."